app/src/main/res/values/strings.xml

```diff
@@ -1,10 +1,20 @@
 <resources>
     <string name="app_name">Attendance Tracker</string>
+
+    <!-- TextViews Text -->
     <string name="welcome">Welcome</string>
     <string name="checked_in">Checked in:</string>
     <string name="arrived_at">Arrived at:</string>
     <string name="checked_out">Checked out:</string>
     <string name="left_at">Left at:</string>
     <string name="total_working_hours">Total working hours</string>
     <string name="today_s_log">Today\'s Log</string>
+    <string name="msg_connected">Connected to Database</string>
+    <string name="msg_disconnected">Disconnected of Database</string>
+    <string name="msg_wifi_closed">Open your WI-FI</string>
+    <string name="yes">Yes</string>
+    <string name="no">No</string>
+    <string name="log_me_in">Login</string>
```

Comment:
Try to add more specific names for the Strings resources.
As later when you go back to you code you might forget what resource related to what, as well as other programmers who read your code, might not understand it.

_For example:_

`<string name="dlg_confirmation _msg_yes">Yes</string>`